namenode/src/main/java/org/apache/crail/namenode/NameNode.java

```diff
@@ -81,8 +81,15 @@ public static void main(String args[]) throws Exception {
 		CrailConstants.NAMENODE_ADDRESS = namenode + "?id=" + serviceId + "&size=" + serviceSize;
 		CrailConstants.printConf();
 		CrailConstants.verify();
-		
-		RpcNameNodeService service = RpcNameNodeService.createInstance(CrailConstants.NAMENODE_RPC_SERVICE);
+
+		RpcNameNodeService service;
+
+		if(CrailConstants.ELASTICSTORE) {
```

Comment:
Do we really need this? Wouldn't it be sufficient if we use `CrailConstants.NAMENODE_RPC_SERVICE` and specify the elastic namenode rpc service class there?